doc/worker.md

```diff
@@ -77,6 +78,26 @@ a [WorkerForkedEvent](../src/Event/WorkerForkedEvent.php) is dispatched.
 
 **PS II: You can determine if the current worker is a fork via the option `isForked`.**
 
+## Using fibers
+
+_Introduced in `0.9`_
+
+Since PHP `8.1`, you can use fibers to "fork" the current process and create an isolated process
```

Comment:
Unfortunately, Fibers are not a new syntax for process fork. The fibers similar to Generators and executed in the same process. 

See RFC https://wiki.php.net/rfc/fibers
> Since fibers exist within a single process thread, switching between fibers is significantly more performant than switching between processes or threads.

Reply:
Hi @GromNaN 👋🏻 

Nice catch, I opened #238 if you want to gave more informations / feedbacks on this one 🙂